### PR TITLE
Update dependencies in autowire branch

### PR DESCRIPTION
**Pull request by stewSquared**

### CODE REVIEW

File: example/jvm/src/main/scala/example/Server.scala [hunk 1]
```diff
@@ -50,19 +53,20 @@ object Server extends SimpleRoutingApp with Api{
       } ~
       post {
         path("api" / Segments){ s =>
-          extract(_.request.entity.asString) { e =>
+          extract(_.request.entity.toString) { e =>
```
Comment (stewSquared) [inline example/jvm/src/main/scala/example/Server.scala]: This is probably a bug right here.

**Comment by stewSquared** [comment]
Don't merge this yet! I need to verify it works!

**Comment by stewSquared** [comment]
Alright! I "fixed" the deserialization.

File: example/jvm/src/main/scala/example/Server.scala [hunk 2]
```diff
@@ -50,19 +53,23 @@ object Server extends SimpleRoutingApp with Api{
       } ~
       post {
         path("api" / Segments){ s =>
-          extract(_.request.entity.asString) { e =>
+          extract(_.request.entity match {
```
Comment (lihaoyi) [inline example/jvm/src/main/scala/example/Server.scala:56]: @stewSquared I think this can be simplified as `entity(as[String]) { e =>`

Comment (stewSquared) [reply example/jvm/src/main/scala/example/Server.scala:56]: @lihaoyi I must be missing an import; that doesn't work for me.
I searched all over the source and couldn't find an equivalent to the old `.asString` method.

**Pull request merged**